## **Superstore-Sales Analysis**

<span style="color: rgb(60, 64, 67); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);">The&nbsp;</span> <span style="border: 0px; font-family: Inter, sans-serif; font-variant-alternates: inherit; font-variant-numeric: inherit; font-variant-east-asian: inherit; font-weight: 700; margin: 0px; padding: 0px; vertical-align: baseline; color: rgb(60, 64, 67); background-color: rgb(255, 255, 255);">"Superstore Sales"</span> <span style="color: rgb(60, 64, 67); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);">&nbsp;dataset is a comprehensive and versatile collection of data that provides valuable insights into sales, customer behavior, and product performance.&nbsp;</span> 

**This notebook has created through Azure Data Studio and provides a Sales Analysis using PostgreSql**

 <span style="color: rgba(0, 0, 0, 0.7); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);">&nbsp;The analysis aims to uncover valuable patterns, trends, and insights that can help the company better understand its sales dynamics, customer behavior, and profitability.</span>  

### Content

The dataset contains records of successful orders. It also contains features such as the `Order Date`, `Ship Date`, `Country` and `Region` from where the customer lives/resides, product `Category`,`Quantity`, `Sales` and `Profit`.

Below are variable descriptions for each of the columns:

- `row_id`: unique row identifier
- `order_id`: unique order identifier
- `order_date`: date the order was placed
- `ship_date`: date the order was shipped
- `ship_mode`: how the order was shipped
- `customer_id`: unique customer identifier
- `customer_name`: customer name
- `segment`: segment of product
- `country`: country of customer
- `city`: city of customer
- `state`: state of customer
- `postal_code`: postal code of customer
- `region`: Superstore region represented
- `product_id`: unique product identifier
- `category`: category of product
- `sub_category`: subcategory of product
- `product_name`: name of product
- `sales`: total sales of that product in the order
- `quantity`: total units sold of that product in the order
- `discount`: percent discount applied for that product in the order
- `profit`: total profit for that product in the order (net profit, all expenses including discount)

Although SQL is used to retrieve data which is then formatted for presentations using other tools ,but my aim is to make this notebook self contained ,for the purpose of which I have formatted table columns for example as currency provided data visualization of the queries

## **Importing the data into PostgreSQL**

In [3]:
DROP TABLE IF EXISTS Superstore_sales;

CREATE TABLE Superstore_sales (
    Row_ID INT,
    Order_ID VARCHAR,
    Order_Date DATE,
    Ship_Date DATE,
    Ship_Mode VARCHAR,
    Customer_ID VARCHAR,
    Customer_Name VARCHAR,
    Segment VARCHAR,
    Country VARCHAR,
    City VARCHAR,
    State VARCHAR,
    Postal_Code INT,
    Region VARCHAR,
    Product_ID VARCHAR,
    Category VARCHAR,
    Sub_Category VARCHAR,
    Product_Name VARCHAR,
    Sales DECIMAL,
    Quantity INT,
    Discount DECIMAL,
    Profit DECIMAL
);


--Although I have created table as ecommerce_Dat Superstore_sales in PostgreSQL and import it here ,so further I am using Superstore_sales Only

Commands completed successfully

Total execution time: 00:00:01.212

In [1]:
--Look at first few records only

Select * 
from superstore_sales
limit 10;

(10 row(s) affected)

Total execution time: 00:00:00.010

row_id order_id order_date ship_date ship_mode customer_id customer_name segment country city state postal_code region product_id category sub_category product_name sales quantity discount profit 1 CA-2013-152156 2013-11-09 2013-11-12 Second Class CG-12520 Claire Gute Consumer United States Henderson Kentucky 42420 South FUR-BO-10001798 Furniture Bookcases Bush Somerset Collection Bookcase 261.96 2 0 41.9136 2 CA-2013-152156 2013-11-09 2013-11-12 Second Class CG-12520 Claire Gute Consumer United States Henderson Kentucky 42420 South FUR-CH-10000454 Furniture Chairs Hon Deluxe Fabric Upholstered Stacking Chairs, Rounded Back 731.94 3 0 219.582 3 CA-2013-138688 2013-06-13 2013-06-17 Second Class DV-13045 Darrin Van Huff Corporate United States Los Angeles California 90036 West OFF-LA-10000240 Office Supplies Labels Self-Adhesive Address Labels for Typewriters by Universal 14.62 2 0 6.8714 4 US-2012-108966 2012-10-11 2012-10-18 Standard Class SO-20335 Sean O'Donnell Consumer United States Fort Lauderdale Florida 33311 South FUR-TA-10000577 Furniture Tables Bretford CR4500 Series Slim Rectangular Table 957.5775 5 0.45 -383.031 5 US-2012-108966 2012-10-11 2012-10-18 Standard Class SO-20335 Sean O'Donnell Consumer United States Fort Lauderdale Florida 33311 South OFF-ST-10000760 Office Supplies Storage Eldon Fold 'N Roll Cart System 22.368 2 0.2 2.5164 6 CA-2011-115812 2011-06-09 2011-06-14 Standard Class BH-11710 Brosina Hoffman Consumer United States Los Angeles California 90032 West FUR-FU-10001487 Furniture Furnishings Eldon Expressions Wood and Plastic Desk Accessories, Cherry Wood 48.86 7 0 14.1694 7 CA-2011-115812 2011-06-09 2011-06-14 Standard Class BH-11710 Brosina Hoffman Consumer United States Los Angeles California 90032 West OFF-AR-10002833 Office Supplies Art Newell 322 7.28 4 0 1.9656 8 CA-2011-115812 2011-06-09 2011-06-14 Standard Class BH-11710 Brosina Hoffman Consumer United States Los Angeles California 90032 West TEC-PH-10002275 Technology Phones Mitel 5320 IP Phone VoIP phone 907.152 6 0.2 90.7152 9 CA-2011-115812 2011-06-09 2011-06-14 Standard Class BH-11710 Brosina Hoffman Consumer United States Los Angeles California 90032 West OFF-BI-10003910 Office Supplies Binders DXL Angle-View Binders with Locking Rings by Samsill 18.504 3 0.2 5.7825 10 CA-2011-115812 2011-06-09 2011-06-14 Standard Class BH-11710 Brosina Hoffman Consumer United States Los Angeles California 90032 West OFF-AP-10002892 Office Supplies Appliances Belkin F5C206VTEL 6 Outlet Surge 114.9 5 0 34.47

<span style="color: rgba(0, 0, 0, 0.7); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);">The dataset contains 9994 records (rows) and 21 features (columns). Among the features, 2 have datetime data type (date), 3 are floating point (decimals), 3 are integers (whole numbers), and 13 are object (strings) data types. It also has no missing values.</span>

**Total Sales Analysis**

In [4]:
--What is the total sales revenue ?

Select SUM(sales) as Total_Sales
from superstore_sales;

(1 row(s) affected)

Total execution time: 00:00:00.019

total_sales 2297200.8603

In [1]:
--Total Orders Placed 

Select COUNT(Distinct Order_ID) as Total_OrdersPlaced
from superstore_sales;

(1 row(s) affected)

Total execution time: 00:00:00.094

total_ordersplaced 5009

In [2]:
--Orders Placed Each Month 

Select Extract(Month from Order_Date) as Month, COUNT (*) as OrdersPlaced 
from superstore_sales
Group by Extract(Month from Order_Date)
Order by Month;


--In the month of Nov most of the sales has been placed 

(12 row(s) affected)

Total execution time: 00:00:00.053

month ordersplaced 1 387 2 301 3 680 4 663 5 750 6 709 7 708 8 710 9 1386 10 817 11 1462 12 1421

In [3]:
--Avg Shipping Duration (Ship date - Order Date)

Select AVG(Ship_Date - Order_Date)
from Superstore_sales;

(1 row(s) affected)

Total execution time: 00:00:00.013

avg 3.9588753251951171

In [2]:
--Customer Insights 

--Top 10 Customers by Total Purchase Amount

Select Customer_Name,Sum(Sales) as Total_Purchase
from superstore_sales
Group by Customer_Name
Order by Total_Purchase
Limit 10;

(10 row(s) affected)

Total execution time: 00:00:00.049

customer_name total_purchase Thais Sissman 4.833 Lela Donovan 5.304 Carl Jackson 16.52 Mitch Gastineau 16.739 Roy Skaria 22.328 Sean O Donnell 30.352 Susan Gilcrest 47.946 Ricardo Emerson 48.36 Larry Blacks 50.188 Adrian Shami 58.82

In [1]:
--Top 10 Customers 

--this Pie Charts helps me 

Select Customer_Name,Sum(Sales) as Total_Purchase
from superstore_sales
Group by Customer_Name
Order by Total_Purchase
Limit 10;

--The pie chart illustrates the division of total purchases among various individuals, providing insights into their respective contributions to overall purchases.

--Adrian stands out with the highest total purchase amount, indicating a significant contribution to the total.
--On the other hand, Thais exhibits the lowest total purchase, representing a comparatively smaller share of the overall purchases.

(10 row(s) affected)

Total execution time: 00:00:00.054

customer_name total_purchase Thais Sissman 4.833 Lela Donovan 5.304 Carl Jackson 16.52 Mitch Gastineau 16.739 Roy Skaria 22.328 Sean O Donnell 30.352 Susan Gilcrest 47.946 Ricardo Emerson 48.36 Larry Blacks 50.188 Adrian Shami 58.82

In [29]:
--Average Order Value per Customer 

Select Customer_Name , AVG(Sales) as AvgOrderValue
from superstore_sales
Group by Customer_Name 
ORDER BY AvgOrderValue DESC
limit 10;



--If I have to find the Max value
Select * from (Select Customer_Name , AVG(Sales) as AvgOrderValue
from superstore_sales
Group by Customer_Name ) as CustomerAvgOrder
ORDER BY AvgOrderValue Desc
limit 1;

--Mitch Willingham is the highest Avgordervalue


--Min Value 

Select * from (Select Customer_Name , AVG(Sales) as AvgOrderValue
from superstore_sales
Group by Customer_Name ) as CustomerAvgOrder
ORDER BY AvgOrderValue
limit 1;


(10 row(s) affected)

(1 row(s) affected)

(1 row(s) affected)

Total execution time: 00:00:00.079

customer_name avgordervalue Mitch Willingham 1751.2920000000000000 Sean Miller 1669.5366666666666667 Tamara Chand 1587.6848333333333333 Grant Thornton 1558.5353333333333333 Tom Ashbrook 1459.5620000000000000 Hunter Lopez 1170.2998181818181818 Bill Shonely 1166.8503333333333333 Christopher Conant 1102.6429090909090909 Stefanie Holloman 1048.1960000000000000 Christopher Martinez 895.4020000000000000

customer_name avgordervalue Mitch Willingham 1751.2920000000000000

customer_name avgordervalue Thais Sissman 2.4165000000000000

In [34]:
--Top Selling Products by Category Sold 

Select Category,Product_Name,Sum(Quantity) as TotalQuantitySold
from Superstore_sales
GROUP by Category,Product_Name
Order by TotalQuantitySold Desc
Limit 10;




(10 row(s) affected)

Total execution time: 00:00:00.106

category product_name totalquantitysold Office Supplies Staples 814 Furniture KI Adjustable-Height Table 74 Office Supplies Storex Dura Pro Binders 71 Office Supplies Avery Non-Stick Binders 71 Office Supplies GBC Premium Transparent Covers with Diagonal Lined Pattern 67 Furniture Situations Contoured Folding Chairs, 4/Set 64 Furniture Staples 62 Furniture Eldon Wave Desk Accessories 61 Furniture Chromcraft Round Conference Tables 61 Furniture Global Wood Trimmed Manager's Task Chair, Khaki 59

In [35]:
--Sales and Profit VS Discount Rate ANalysis 

Select Discount, Avg(Sales) as Avgsales,AVG(Profit) as AVGProfit
from superstore_sales
GROUP by Discount;

(12 row(s) affected)

Total execution time: 00:00:00.036

discount avgsales avgprofit 0.3 454.7429735682819383 -45.6796361233480176 0.15 529.9715673076923077 27.2882980769230769 0.8 56.5458533333333333 -101.7967973333333333 0.2 209.0769395679518731 24.7025720535958436 0.5 892.7051515151515152 -310.7034560606060606 0.45 498.6340000000000000 -226.6464636363636364 0.32 536.7947703703703704 -88.5606555555555556 0.6 48.1500000000000000 -43.0772115942028986 0 226.7420737807419758 66.9002924551896624 0.7 97.1777081339712919 -95.8740595693779904 0.1 578.3973510638297872 96.0550744680851064 0.4 565.1348737864077670 -111.9274291262135922

In [42]:
--What are the sales figures for each State in United States ?

Select State,Sum(Sales) as Total_Sales
from superstore_sales
Group BY State
Limit 10;

(10 row(s) affected)

Total execution time: 00:00:00.008

state total_sales Oklahoma 19683.39 Colorado 32108.118 North Carolina 55603.164 Mississippi 10771.34 Florida 89473.7080 Vermont 8929.37 Delaware 27451.069 Nevada 16729.102 Louisiana 9217.03 New York 310876.271

**This bar chart visually represents the total sales figures for different states, highlighting California with the highest sales and North Dakota with the lowest sales.**

**California: Highest Sales**

**California stands out with the maximum sales figures among all states in the dataset, indicating robust sales performance.**

**North Dakota: Lowest Sales**

**North Dakota represents the state with the minimum sales figures, reflecting comparatively lower sales compared to other states.**

**The bar chart provides a clear visual comparison of total sales across states, emphasizing the contrast between the highest and lowest sales performers among the represented states.**

In [45]:


Select State,Sum(Sales) as Total_Sales
from superstore_sales
Group BY State;

(49 row(s) affected)

Total execution time: 00:00:00.011

state total_sales Oklahoma 19683.39 Colorado 32108.118 North Carolina 55603.164 Mississippi 10771.34 Florida 89473.7080 Vermont 8929.37 Delaware 27451.069 Nevada 16729.102 Louisiana 9217.03 New York 310876.271 West Virginia 1209.824 New Jersey 35764.312 South Carolina 8481.71 Arkansas 11678.13 New Mexico 4783.522 Missouri 22205.15 Connecticut 13384.357 South Dakota 1315.56 District of Columbia 2865.02 Iowa 4579.76 Indiana 53555.36 Massachusetts 28634.434 Rhode Island 22627.956 Ohio 78258.136 Michigan 76269.614 Minnesota 29863.15 Pennsylvania 116511.914 Washington 138641.270 Montana 5589.352 Wisconsin 32114.61 Kentucky 36591.75 Arizona 35282.001 Illinois 80166.101 Virginia 70636.72 Maryland 23705.523 Georgia 49095.84 Wyoming 1603.136 Utah 11220.056 New Hampshire 7292.524 North Dakota 919.91 Maine 1270.53 Nebraska 7464.93 California 457687.6315 Tennessee 30661.873 Kansas 2914.31 Oregon 17431.150 Texas 170188.0458 Idaho 4382.486 Alabama 19510.64

In [47]:
--How Many Customers Purchased product each Month ?

With MonthlyCustomersCounts as (
    Select Extract (Month from Order_Date) as Month,
    COUNT(Distinct Customer_ID) as UniqueCustomers
    from superstore_sales
    GROUP by Extract(Month from Order_Date)
)
Select * from MonthlyCustomersCounts ORDER by MONTH ;

(12 row(s) affected)

Total execution time: 00:00:00.162

month uniquecustomers 1 163 2 145 3 283 4 277 5 309 6 287 7 278 8 277 9 460 10 323 11 481 12 457

In [58]:
--Overall Sales Trend per Year 

Select EXTRACT(Year from Order_Date) as SalesYear ,Sum(Sales) as TotalSales
from superstore_sales
Group by EXTRACT(Year from Order_Date)
Order by SalesYear;

(4 row(s) affected)

Total execution time: 00:00:00.012

salesyear totalsales 2011 484247.4981 2012 470532.5090 2013 608473.8300 2014 733947.0232

(4 row(s) affected)

Total execution time: 00:00:00.021

salesyear totalsales 2011 484247.4981 2012 470532.5090 2013 608473.8300 2014 733947.0232

<span style="color: rgba(0, 0, 0, 0.7); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);">Over time, orders had increased and so are sales. However, a slight dip in sales can be obsered in 2012. From 484,247 dollars total sales in 2011, Superstore sales slightly dipped to 470,532 dollars in the following year, which is a 2.83% difference or 13,715 dollars.</span>

In [59]:
-- Total Sales Per Sub Category 

Select Sub_Category,Sum(Sales) as TotalSales
from superstore_sales
Group by Sub_Category
Order by TotalSales ;

(17 row(s) affected)

Total execution time: 00:00:00.017

sub_category totalsales Fasteners 3024.280 Labels 12486.312 Envelopes 16476.402 Art 27118.792 Supplies 46673.538 Paper 78479.206 Furnishings 91705.164 Appliances 107532.161 Bookcases 114879.9963 Copiers 149528.030 Accessories 167380.318 Machines 189238.631 Binders 203412.733 Tables 206965.5320 Storage 223843.608 Chairs 328449.103 Phones 330007.054

(17 row(s) affected)

Total execution time: 00:00:00.012

sub_category totalsales Fasteners 3024.280 Labels 12486.312 Envelopes 16476.402 Art 27118.792 Supplies 46673.538 Paper 78479.206 Furnishings 91705.164 Appliances 107532.161 Bookcases 114879.9963 Copiers 149528.030 Accessories 167380.318 Machines 189238.631 Binders 203412.733 Tables 206965.5320 Storage 223843.608 Chairs 328449.103 Phones 330007.054

The visualization above shows a general overview of the magnitude of sales for each product sub-category. For technology category, phones are the top sales-generating products. Chairs products for furnitures category, and storage products for office supplies category. 

Throughout the 4-year period from 2011 - 2014, phones, chairs, and storage products are the three most sales-generating products. Along with them are tables, binders, and machine products.

Under the technology category, copier products are the least performing. For the furniture and office supplies category, furnishings and fasteners are the least performing.

In [63]:
--Total Sales Yearly in each Region 

Select EXTRACT(Year from Order_Date) as SalesYear,Region ,Sum(Sales) AS TotalSales
FROM superstore_sales
Group By EXTRACT(Year from Order_Date),Region
Order By SalesYear;

(16 row(s) affected)

Total execution time: 00:00:00.020

salesyear region totalsales 2011 South 103845.8435 2011 West 147883.0330 2011 East 128680.457 2011 Central 103838.1646 2012 West 139966.2495 2012 East 156332.057 2012 Central 102874.2220 2012 South 71359.9805 2013 Central 147429.3760 2013 West 186976.0165 2013 South 93539.1275 2013 East 180529.310 2014 Central 147098.1282 2014 South 122976.9535 2014 West 250632.5255 2014 East 213239.416

Shown is how sales for each region changed over time. In the Central region, a sharp positive growth was observed from 2013, while yearly positive growth was consistent in the East. For the South region, a negative growth was observed in 2012, but the region rebounded thereafter. Simililarly for the West, negative growth was observed in 2012, but rebounded and sustained positive growth thereafter.

It was observed that there was a slight dip in total sales in 2012. From the visualization above, it can be inferred that South had contributed the most to that dip, followed by the West and then the Central. Interestingly, the East region still grew positively in 2012.

In [15]:
--Calculate Profit Margin of each Year 

Select EXTRACT (Year from Order_Date) AS SalesYear,
SUM(Profit) as TotalProfit,
Sum(sales) as TotalSales,
SUM(Profit) / Sum(sales) * 100 as ProfitMargin
from superstore_sales
Group by Extract(Year from Order_Date);



(4 row(s) affected)

Total execution time: 00:00:00.016

salesyear totalprofit totalsales profitmargin 2011 49543.9741 484247.4981 10.23112649923673400200 2012 61618.6037 470532.5090 13.09550403455757825200 2013 81726.9308 608473.8300 13.43146192499355313300 2014 93507.5131 733947.0232 12.74036274339098647900

<span style="color: rgba(0, 0, 0, 0.7); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);">Using profit margin, the company had generated the least profit in 2011 at 10.2311%. After a year, 2012, the company generated relatively higher profit at 13.0955% margin. This continued and the company registered a higher profit margin in 2013 at 13.4315%. The trend, however, slowed down and the company generated a lower profit margin at 12.7404%, even lower than that of 2012.</span>

In [20]:
--Calculate Profit Margin of Category and Sub category 

Select Category,Sub_Category,
Sum(Profit) as TotalProfit,
Sum(Sales) as TotalSales,
Sum(Profit) / Sum(Sales) * 100 as ProfitMargin
from superstore_sales
Group by Category,Sub_Category
Order by Category;

(17 row(s) affected)

Total execution time: 00:00:00.015

category sub_category totalprofit totalsales profitmargin Furniture Chairs 26590.1663 328449.103 8.09567328914276255500 Furniture Bookcases -3472.5560 114879.9963 -3.02276820320545222700 Furniture Furnishings 13059.1436 91705.164 14.24035793665883417400 Furniture Tables -17725.4811 206965.5320 -8.56446043392384776400 Office Supplies Binders 30221.7633 203412.733 14.85736062550223933100 Office Supplies Art 6527.7870 27118.792 24.07108325474084538900 Office Supplies Supplies -1189.0995 46673.538 -2.54769522721847227400 Office Supplies Storage 21278.8264 223843.608 9.50611303584777815100 Office Supplies Envelopes 6964.1767 16476.402 42.26758184220074261400 Office Supplies Paper 34053.5693 78479.206 43.39183719570251513500 Office Supplies Fasteners 949.5182 3024.280 31.39650429193064134300 Office Supplies Appliances 18138.0054 107532.161 16.86751687246385758000 Office Supplies Labels 5546.2540 12486.312 44.41867222283088873600 Technology Machines 3384.7569 189238.631 1.78861836090961786800 Technology Copiers 55617.8249 149528.030 37.19558460042575295100 Technology Accessories 41936.6357 167380.318 25.05469950176579303700 Technology Phones 44515.7306 330007.054 13.48932698874976169400

<span style="color: rgba(0, 0, 0, 0.7); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);">For furnitures products, furnishings products, on average, are the most profitable followed by chairs products. On the other hand, the company was operating at a loss on tables and bookcases products. Assuming similar sales, loss on tables are higher than gains on furnishings.</span>

<span style="color: rgba(0, 0, 0, 0.7); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);"><br></span>

<span style="color: rgba(0, 0, 0, 0.7); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);">On technology products, the company, on average, was profiting more compared to furniture ones. Three of its product sub-categories were generating profit at much higher margin profit: 13% for phones, 25% for tech accesories, and 37% profit margin for copiers products. The company, on the other hand, Machines products, was operating at a loss with machine products.</span><span style="color: rgba(0, 0, 0, 0.7); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);"><br></span>

<span style="color: rgba(0, 0, 0, 0.7); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);"><br></span>

<span style="color: rgba(0, 0, 0, 0.7); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);">Not only do five product sub-categories under office supplies had over 20% profit margin, three of which had over 40% discount, two of the remaining four sub-categories had also positive profit margin of 9% and 11%. The company, on the other hand, were operating at a loss : Supplies products during the 4 year period.</span><span style="color: rgba(0, 0, 0, 0.7); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);"><br></span>

In [29]:
--What Portion of the orders Shipped through each mode ?

Select Ship_Mode,
COUNT(*) as NumberOfOrders,
COUNT(*) * 100.0 / (SELECT COUNT(*) from Superstore_sales) as OrderPercentage
from Superstore_sales
Group by Ship_Mode;


(4 row(s) affected)

Total execution time: 00:00:00.009

ship_mode numberoforders orderpercentage Standard Class 5968 59.7158294976986192 Second Class 1945 19.4616770062037222 Same Day 543 5.4332599559735842 First Class 1538 15.3892335401240744

(4 row(s) affected)

Total execution time: 00:00:00.004

ship_mode numberoforders orderpercentage Standard Class 5968 59.7158294976986192 Second Class 1945 19.4616770062037222 Same Day 543 5.4332599559735842 First Class 1538 15.3892335401240744

<span style="color: rgba(0, 0, 0, 0.7); font-family: Inter, sans-serif; background-color: rgb(255, 255, 255);">60% of all orders had been shipped through standard class mode. 20% was shipped through second class , and 15% was shipped through first class mode. 5% of all orders was shippied through same day mode.</span>

# Conclusion

**Sales Performance:**

The company witnessed consistent year-over-year sales growth, notably in 2013, showcasing the highest increase and 2012 displaying the slowest.

Seasonal trends were noticeable, with sales peaks observed in November, December, and September. Variability was pronounced in March, September, and October.

  

**Top Selling Categories:**

Phones, chairs, and storage items emerged as the leading products within their respective categories.

Conversely, copiers, furnishings, and fasteners displayed lower sales figures.

  

**Regional Sales Analysis:**

The West and East regions exhibited higher sales volumes across most categories, while the South consistently reported lower sales.

In 2012, the Central region showed notable growth, setting it apart from other regions during that year. The West region notably displayed the fastest average annual sales growth.

  

**Profitability Insights:**

Despite fluctuating profitability, the company maintained a consistent profit margin of 10% or higher between 2011 and 2014.

Sub-categories such as chairs, phones, and storage products showed relatively lower profitability.

Conversely, furnishings, copiers, and labels stood out as the most profitable sub-categories.

Discounts offered significantly impacted profit margins, especially affecting tables and office supplies.

  

**Concluding Remarks:**

**While the company exhibited commendable sales growth and sustained profitability, variations in sales due to seasonality and regional disparities played pivotal roles in its overall performance.**

**Profit margins were notably affected by the strategic use of discounts, underscoring their influence on overall profitability.**